### ISSUE TITLE
wtforms 2.3.0 incompatibility

### ISSUE THREAD

**Issue by kprzerwa** [inveniosoftware/invenio-oauth2server#200]
WTForms in 2.3.0 version requires installation of email_validator library, however when done so we have a broken import:
```
 File ".../.virtualenvs/.../lib/python3.8/site-packages/invenio_oauth2server/forms.py", line 16, in <module>
    from wtforms.widgets import HTMLString
ImportError: cannot import name 'HTMLString' from 'wtforms.widgets' (.../.virtualenvs/.../lib/python3.8/site-packages/wtforms/widgets/__init__.py)
```

**Comment by ppanero** [comment]
Related https://github.com/inveniosoftware/invenio-access/issues/172 (WTForms incompatibility also)

**Comment by lnielsen** [comment]
**Problem**
[Travis build error on lowest requirements](https://travis-ci.org/github/inveniosoftware/invenio-oauth2server/jobs/681204492)

WTForms moving email validation to an optional dependency. This problem was fixed in invenio-accounts v1.1.4.

**Task:**
- Bump minimal version of invenio-accounts from v1.1.3 to v1.1.4 in ``setup.py``. This should fix the build error.

**Comment by ppanero** [comment]
Bumped versions of:

- Invenio-accounts.
- Then find error with Sphinx.
- Then found error with requirements builder. <- Current state
